bot/cogs/weather.py

```diff
@@ -0,0 +1,105 @@
+"""Commands for the weather module."""
+import discord
+from discord.ext import commands
+import aiohttp
+from bot.settings import WEATHER_TOKEN
+
+
+class Weather(commands.Cog, name='weather'):
+    """commands for the weather finder."""
+
+    def __init__(self, bot: discord.ext.commands.bot.Bot):
+        self.bot = bot
+
+    @commands.command(
```

Comment:
```suggestion
    @commands.command(brief="Takes in a city name and returns the weather for that location")
```
this could be one line, if it is broken into multiple lines, the `)` should be on a newline as well like this:
```python
@commands.command(
    brief="Takes in a city name and returns the weather for that location"
    )
```